packages/Kokkos/test/tstKokkosView.cpp

```diff
@@ -94,21 +94,19 @@ class AssignFunctor
                        "View ranks must be 2" );
         DTK_REQUIRE( view_1.extent(0) == view_2.extent(0) );
         DTK_REQUIRE( view_1.extent(1) == view_2.extent(1) );
-        _extent = view_1.extent(1);
     }
 
     KOKKOS_INLINE_FUNCTION
     void operator()(const size_t i) const
     {
-        for ( int n = 0; n < _extent; ++n )
+        for ( int n = 0; n < view_1.extent(1); ++n )
```

Comment:
FYI: The loops bounds should not be calculated inside the loop. This would prevent vectorization. It's usually better to pre-calculate the bounds before the loop.